gnomad_qc/v4/annotations/Dockerfile_vrs084_v4

```diff
@@ -34,14 +36,17 @@ RUN apt-get -y update && apt-get install -y --no-install-recommends \
 	# clean up apt cache
     rm -rf /var/lib/apt/lists/*
 
+# Copy in SeqRepo from local file structure
+# Put this big folder (~12G) in the first layers so that it doesn't get rebuilt every time
```

Comment:
If the seqrepo never really changes, it might actually be better to put this directly after the `FROM 'google/cloud-sdk:slim'` line. The packages that get installed with `apt-get` change with some regularity, probably every few weeks or so.